examples/twittermap/web/conf/application.conf

```diff
@@ -146,4 +147,11 @@ liveTweetQueryInterval = 60
 #                             how many seconds before now is the query's end time condition. (Default: 30)
 liveTweetQueryOffset = 30
 # Live Tweet update rate - Every how many seconds, we popup a tweet from queue and show it on page. (Default: 2)
-liveTweetUpdateRate = 2
\ No newline at end of file
+liveTweetUpdateRate = 2
+# In order to enable live tweets feature, 
+# please refer to https://developer.twitter.com/en/docs/basics/authentication/guides/access-tokens.html to get your own twitter developer access keys and tokens and fill them to the following 4 parameters.
+liveTweetConsumerKey = ""
+liveTweetConsumerSecret = ""
+liveTweetToken = ""
+liveTweetTokenSecret = ""
+enableLiveTweet = true
```

Comment:
Since by default the `token`s are all empty, `enableLiveTweets` should be `false`?